pipeline/transformer/transformer_common/src/main/scala/weco/catalogue/transformer/TransformerWorker.scala

```diff
@@ -89,11 +97,38 @@ trait TransformerWorker[Payload <: SourcePayload, SourceData, SenderDest]
         StoreReadError(err, p)
       }
 
+  private def compareToStored(workResult: Result[(Work[Source], StoreKey)]): Future[Result[(Option[Work[Source]], StoreKey)]] =
+    workResult match {
+
+      // Once we've transformed the Work, we query forward -- is this a work we've
```

Comment:
This is really neat way of doing the comparison that we can do now we store the work at this point.

Reply:
The other neat thing about it is that the "interesting" data is pushed into WorkData, so the comparison is easy.

_"Are these two case classes the same"_ vs _"Are these two case classes the same (except this one field)"_